pkg/lockutil/lockutil_unix.go

```diff
@@ -1,5 +1,4 @@
 //go:build !windows
```

Comment:
Probably this line is not needed

Reply:
Nope. It's required because of the error https://github.com/lima-vm/lima/pull/1967#issuecomment-1785459226